modules/ROOT/pages/writing.adoc

```diff
@@ -52,9 +52,10 @@ To persist data into Neo4j, the Spark Connector supports two save modes that
 work only if `UNIQUE` or `NODE KEY` constraints are defined in Neo4j for the given properties.
 
 [NOTE]
+===
```

Comment:
```suggestion
====
```

Reply:
Otherwise, this happens:

<img width="864" alt="Screenshot 2024-02-08 at 11 42 18" src="https://github.com/neo4j/docs-spark/assets/1252064/6d210d6d-3064-4d67-9757-40608729ec7b">